scripts/install-toolchain.sh

```diff
@@ -18,13 +18,16 @@ fi
 
 case $(uname -s) in
   Darwin)
-    toolchain_download="$swift_tag-osx.tar.gz"
+    toolchain_download="$swift_tag-macos-x86_64.tar.gz"
+    echo '-macos' >> .swift-version
```

Comment:
Do we still need this line?

Reply:
Still waiting for the new 5.3 toolchain to be tagged tonight that fixes this.

Reply:
That's cleaned up now, ready for review.